Solutions/Lookout/Workbooks/LookoutEvents.json

```diff
@@ -4,7 +4,7 @@
     {
       "type": 1,
       "content": {
-        "json": ">**NOTE:** This workbook uses a parser based on a Kusto Function to normalize fields. [Follow these steps](https://aka.ms/sentinel-lookoutapi-parser) to create the Kusto function alias **LookoutEvents**."
+        "json": ">**NOTE:** This workbook uses a parser based on a Kusto Function to normalize fields. [Follow these steps](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Lookout/Parsers/LookoutEvents.txt) to create the Kusto function alias **LookoutEvents**."
```

Comment:
I have created short link for parser. https://aka.ms/sentinel-lookoutapi-parser. Please revert this changes. Thanks!